dtfs-central-api/src/services/portal/deal.service.add-gef-deal-cancelled-activity.test.ts

```diff
@@ -34,36 +42,29 @@ describe('PortalDealService - addGefDealCancelledActivity', () => {
         dealType: DEAL_TYPE.GEF,
         author,
         auditDetails,
+        effectiveFrom,
       });
 
       // Assert
       expect(addPortalActivityMock).toHaveBeenCalledTimes(1);
-
-      const expectedActivity = {
-        label: PORTAL_ACTIVITY_LABEL.DEAL_CANCELLED,
-        timestamp: getUnixTime(new Date()),
-        author: {
-          _id: author._id,
-          firstName: UKEF.ACRONYM,
-        },
-      };
-
       expect(addPortalActivityMock).toHaveBeenCalledWith(dealId, expectedActivity, auditDetails);
     });
   });
 
-  describe(`when dealType is not ${DEAL_TYPE.GEF}`, () => {
+  describe(`when deal type is not ${DEAL_TYPE.GEF}`, () => {
     it('should not call addPortalActivity', async () => {
       // Act
       await PortalDealService.addGefDealCancelledActivity({
         dealId,
         dealType: DEAL_TYPE.BSS_EWCS,
         author,
         auditDetails,
+        effectiveFrom,
       });
 
       // Assert
       expect(addPortalActivityMock).toHaveBeenCalledTimes(0);
+      expect(addPortalActivityMock).not.toHaveBeenCalledWith(dealId, expectedActivity, auditDetails);
```

Comment:
do we need this test if the one above is testing if been called 0 times?

Reply:
Good point have removed called with.